docs/api/sandbox-option.md

```diff
@@ -1,9 +1,9 @@
 # `sandbox` Option
 
-> Create a browser window with renderer that can run inside chromium OS sandbox.
+> Create a browser window with a renderer that runs inside the Chromium OS sandbox.
```

Comment:
I've updated the summary to be more comprehensive: https://github.com/electron/electron/pull/9054/commits/92d33995372510b2a6c93c4aa7bda5860ecfa3d2

As an aside, I do wonder if there is much value in using `sandbox: true` without `--enable-sandbox`? And the very existence of this option makes it quite easy to accidentally think that you have the Chromium sandbox enabled when in fact it isn't.

Perhaps the property should be renamed `sandbox-compatible: true`?